lession_1/js/Employee.js

```diff
@@ -0,0 +1,12 @@
+class Employee extends Human {
+    constructor(salary, department) {
+        super();
```

Comment:
```suggestion
        super();
```
здесь вы инициализируете свойства `Human`, но вы не передаете какие либо данные, было бы уместно указать у конструктора `constructor(name, age, dateOfBirth, salary, department)`, а ниже прописать `super(name, age, dateOfBirth)`